src/webviews/extension-side/dataviewer/dataViewerCommandRegistry.ts

```diff
@@ -147,7 +147,7 @@ export class DataViewerCommandRegistry implements IExtensionSingleActivationServ
         }
 
         if (pythonPath) {
-            let untildePath = debugConfiguration.python;
+            let untildePath = pythonPath;
```

Comment:
Just a mess up on my part when I touched this last :(. The goal was to not look at .python anymore as it's often, but not always the path. But I accidentally left this in. 

I added an E2E test for the positive case here already, and I don't see a need for a specific regression tests as it would just be passing in a malformed .python, which at this point isn't any different from any other property on the debugConfiguration which we don't look at.